NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -115,6 +115,35 @@ public class LitecoinNodeDownloadData
 			};
 		}
 
+		public class ViacoinNodeDownloadData
```

Comment:
you forgot to expose it as a property as well. Scroll down this file to see how.